setup.py

```diff
@@ -6,6 +6,11 @@
 
 MINIMUM_CYTHON_VERSION = '0.13'
 
+
+def cmp(a, b):
```

Comment:
@axiak  At https://github.com/axiak/pyre2/blob/master/setup.py#L26. `cmp` function is not there in Python 3; so this is to replace it.